README.md

```diff
@@ -14,29 +14,34 @@ directly from single 3'end RNA sequencing data.
 ## Requirements
 ![](directory.png)
 
-1) installation of nextflow.
+1) installation of nextflow and dependencies.
+
+```bash
+mamba create -n scinpas nextflow
```

Comment:
Info: create environment called scinpas and put nextflow into this environment
next time, when you want to use it, type conda activate scinpas 
then it will activate scinpas environment.